src/avfoundation.cs

```diff
@@ -9269,15 +9269,16 @@ interface AVAssetExportPresetApple {
 
 		[Field ("AVAssetExportPresetAppleM4V720pHD")]
 		NSString M4V720pHD { get; }
-
+#else 
+		[NoiOS]
 		[Field ("AVAssetExportPresetAppleM4V1080pHD")]
 		NSString M4V1080pHD { get; }
```

Comment:
This means you're removing `M4V1080pHD` and `ProRes422Lpcm` from macOS, which is a breaking change.

Any particular reason why you can't remove the `#if MONOMAC` completely, and add `[NoXYZ]` to each field?

Reply:
I'm stupid, that should have been and #endif but is better with the NoXYZ attrs